shared/pkg/scanner/grype/local_grype.go

```diff
@@ -74,18 +74,11 @@ func (s *LocalScanner) run(sourceType utils.SourceType, userInput string) {
 	var hash string
 	origInput := userInput
 	if sourceType == utils.SBOM {
-		syftJSONFilePath, cleanup, err := ConvertCycloneDXFileToSyftJSONFile(userInput, s.logger)
```

Comment:
`ConvertCycloneDXFileToSyftJSONFile` is used also in `remote_grype` - we need to remove it also in that case and fix the code in the grype server to be able to handle any sbom formt?

Reply:
removed the conversion from remote_grype too, thanks!